app/api/exports.py

```diff
@@ -283,6 +344,11 @@ def export_speakers_pdf(event_identifier):
     else:
         event_id = event_identifier
 
+    if not has_access('is_coorganizer', event_id=event_id):
+        return abort(
+            make_response(jsonify(error="Access Forbidden"), 403)
+        )
```

Comment:
Make a function assert_forbidden which raises Forbidden Error if passwd argument is False